src/base/PrecomputedCodeContext.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 1996-2024 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID_SRC_BASE_PRECOMPUTEDCODECONTEXT_H
+#define SQUID_SRC_BASE_PRECOMPUTEDCODECONTEXT_H
+
+#include "base/CodeContext.h"
+#include "base/InstanceId.h"
+#include "sbuf/SBuf.h"
+
+/// CodeContext with constant details known at construction time
```

Comment:
IIRC, compiling this header code requires including `<ostream>` because this code uses "<<" expressions.  Please include that STL header even if it is currently included (directly or indirectly) by the headers above.

```suggestion
#include <ostream>

/// CodeContext with constant details known at construction time
```

IIRC, including `<iosfwd>` is not enough in such cases.

Reply:
Done.